presto-main/src/test/java/io/prestosql/execution/TestSetSessionTask.java

```diff
@@ -57,6 +57,7 @@
 import static java.lang.String.format;
 import static java.util.Collections.emptyList;
 import static java.util.concurrent.Executors.newCachedThreadPool;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.fail;
```

Comment:
Please remove this unused import.

Reply:
done

Reply:
@somayaj Thanks, please squash commits into one.

Reply:
done